experiment/schedule_measure_workers.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for starting instances to run measure workers."""
+import collections
+import os
+import posixpath
+import sys
+import time
+
+from common import experiment_utils
+from common import gce
+from common import gcloud
+from common import logs
+from common import queue_utils
+from common import yaml_utils
+
+logger = logs.Logger('schedule_measure_workers')  # pylint: disable=invalid-name
+
+# This is the default quota on GCE.
+# TODO(metzman): Use the GCE API to determine this quota.
```

Comment:
Is this todo really needed ? gce instance manager limit is 4000 and can be hardcoded.

Reply:
I think our quota is 1000, and we should have this TODO because hardcoding will break open source users right?